modules/misc/version.nix

```diff
@@ -14,6 +14,7 @@ with lib;
         "21.03"
         "21.05"
         "21.11"
+        "22.05"
       ];
       default = "18.09";
```

Comment:
NixOS updates the default `stateVersion` every release, maybe we should do the same?

Reply:
Hmm, I didn't know that. I'm somewhat against that since it could cause issues if you forget to set it. That said, it seems NixOS has done this for a long time so I imagine either very few state problems happen in practice or very few omit stateVersion from their configuration.

I would be more inclined to remove the default altogether to force the remaining few people without an explicit state version to set it 🙂

Not sure which is the best option, will need some pondering. I agree that keeping the default as-is forever seems like a bad idea.